resources/schema.gql

```diff
@@ -164,7 +164,19 @@ type CreateDatasetResultSuccess implements CreateDatasetResult & CreateDatasetFr
 }
 
 type CronExpression {
-	cronExpression: String!
+	min: String!
```

Comment:
My preference would be to leave cron parameter a string and instead specify in the documentation:
- the formats that we support
  - 5-term with minutes
  - 6-term with seconds
- add a note that we don't guarantee any timings and that all scheduling is best-effort

 If you decide to proceed with this structure my only suggestion is to use `minute` instead of `min`, as all other terms have full names already.